composer.json

```diff
@@ -3,16 +3,65 @@
   "description": "Apache Avro™ is a data serialization system.",
   "minimum-stability": "stable",
   "license": "Apache-2.0",
+  "homepage": "http://avro.apache.org",
+  "type": "library",
+  "keywords": [
+    "RPC"
```

Comment:
Is this the correct keyword to use for avro?

Reply:
https://github.com/apache/avro-rs/blob/0868a8b2d6aa90070bcb6a3388555746d3e44546/Cargo.toml#L37 - these are the keywords used for the Rust SDK

Reply:
Updated to match. Thanks!